client/src/components/SearchInput.js

```diff
@@ -1,16 +1,96 @@
 import React from 'react'
-import { Box, Button, TextInput } from 'grommet'
+import { Box, Button, Select, TextInput, Keyboard } from 'grommet'
+import { useSearchResources } from '../hooks/useSearchResources'
+import { Mappings } from './resources'
+import { sortedObjectKeys } from '../helpers/sortObjectKeys'
+import { getReadable } from '../helpers/readableNames'
+
+export default function SearchInput({
+  initialString = '',
+  onChange,
+  size = 'medium'
+}) {
+  const {
+    query: { search },
```

Comment:
Can this variable be `searchString`?

Reply:
this is to match the API
https://github.com/AlexsLemonade/resources-portal/blob/dev/api/resources_portal/views/document_views.py#L143

Reply:
i changed it to be obvious that it is for the query object to `query.search` the other option was rewriting it but id rather have it match